### DISCUSSION TITLE
Printed view does not apply responsive styles

### DISCUSSION THREAD

**Discussion by farhan27555** [tailwindlabs/tailwindcss#14309] [Help]
Consider this case:

```
<p class="lg:hidden">
    Hello
</p>
```

When the user visits the page with a device with screen size larger than `lg`. They won't see the text `hello`.
However, if the user tries to print the page, the print still includes the text `Hello`. The issue is caused when we use the responsive style prefixes like `lg:`

**Comment by wongjn** [comment]
You could consider using the [`print` variant](https://tailwindcss.com/docs/hover-focus-and-other-states#print-styles) to apply CSS specifically for printing.
